features/javascript-modules.md

```diff
@@ -0,0 +1,15 @@
+---
+title: JavaScript Modules
+category: js
+bugzilla: 1240072
+firefox_status: 60
+mdn_url:
```

Comment:
There's still nothing about this on MDN?

Reply:
The closest we have right now is https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import (we also have a page for export). We need to add a proper guide to it, which is one of my next tasks.

You could link to this: https://hacks.mozilla.org/2015/08/es6-in-depth-modules/, which is really good. It's neerly MDN ;-)